db.go

```diff
@@ -63,6 +70,12 @@ func (m Mysql) MigrationLogDeleteSql() string {
 	return "DELETE FROM gomigrate WHERE migration_id = ?"
 }
 
+func (m Mysql) GetMigrationCommands(sql string) []string {
+	count := strings.Count(sql, ";")
+	commands := strings.SplitN(string(sql), ";", count)
```

Comment:
Although this might help your case, it might not be entirely valid.  See: https://github.com/mattes/migrate/issues/38